nemo/collections/nlp/data/language_modeling/megatron/gpt_dataset.py

```diff
@@ -162,7 +183,19 @@ def get_indexed_dataset_(data_prefix, data_impl, skip_warmup):
 
 
 class GPTDataset(MegatronDataset):
-    def __init__(self, cfg, trainer, name, data_prefix, documents, indexed_dataset, num_samples, seq_length, seed):
+    def __init__(
```

Comment:
@MaximumEntropy Could you look at this updated `GPTDataset` as T5 dataset inherits?

Reply:
@crcrpar I can create a PR to this branch that fixes this for T5LMAdaptedDataset.

Reply:
Does that mean the change for T5LMAdaptedDataset in this PR isn't enough?

Reply:
No, it's good. I didn't realize you already made the change by the time I commented. I was still looking at the LGTM link you shared.